CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.46.0      2022-10-21
+
+* Fix bug collecting the relation stats with child tables with locks
```

Comment:
Maybe "Relation stats: Skip statistics collection on child tables when parent is locked"?